scylla/tests/ccm-integration/test_example.rs

```diff
@@ -0,0 +1,128 @@
+use crate::cluster::{Cluster, ClusterOptions};
+use anyhow::{Context, Error};
+use itertools::Itertools;
+use lazy_static::lazy_static;
+use scylla::client::session::Session;
+use scylla::client::session_builder::SessionBuilder;
+use std::ops::Deref;
+use std::sync::Arc;
+use tokio::sync::RwLock;
+
+lazy_static! {
+    static ref CLUSTER_VERSION: String =
+        std::env::var("SCYLLA_TEST_CLUSTER").unwrap_or("release:6.2.2".to_string());
+    static ref TEST_KEEP_CLUSTER_ON_FAILURE: bool = !std::env::var("TEST_KEEP_CLUSTER_ON_FAILURE")
+        .unwrap_or("".to_string())
+        .parse::<bool>()
+        .unwrap_or(false);
+}
+
```

Comment:
The files with tests should not interact with environment at all. Cluster version selection (and thus reading the env var) should be done by the ccm utils.

Preferably, all interaction with env variables should be done in one place in the code.